sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -733,3 +733,19 @@ def _deserialize_throughput(throughput: list) -> Any:
         offer_throughput=throughput,
         properties=throughput_properties[0]
     )
+
+
+def _replace_throughput(throughput: Union[int, ThroughputProperties], new_throughput_properties: list):
+    max_throughput = throughput.auto_scale_max_throughput
+    increment_percent = throughput.auto_scale_increment_percent
+
+    if isinstance(throughput, object):
```

Comment:
Don't use this - you're already access the attributes on line 739.
This should be using the same kind of logic as your `set_throughput_options` above
Instead try something like:
```python
try:
    if throughput.auto_scale_max_throughput is not None:
        ...
except AttributeError:
    if isinstance(throughput, int)
        ...
    else:
        raise TypeError(...)
```